modules/ui_common.py

```diff
@@ -36,6 +36,29 @@ def plaintext_to_html(text, classname=None):
     return f"<p class='{classname}'>{content}</p>" if classname else f"<p>{content}</p>"
 
 
+def update_logfile(logfile_path, fields):

```

Comment:
I'd recommend using typehints whenever adding new functions. The repo doesn't have them now but it would definitely be nice if new functions were implemented with them for functions/classes.

Also a simple docstring just to satisfy linters wouldn't hurt. This is just an example description of the function so if you can think of a better one then feel free to modify it.

```diff
-def update_logfile(logfile_path, fields):
+def update_logfile(logfile_path: str, fields: list[str]) -> None:
+    """Updates existing log file rows to match current log file column count."""
```

Reply:
agreed, I mostly wanted to retain the style of the surrounding, but if @AUTOMATIC1111 is in agreement, I'd be happy to add the typehints